spec/controllers/conference_registration_controller_spec.rb

```diff
@@ -0,0 +1,255 @@
+require 'spec_helper'
+
+describe ConferenceRegistrationsController, type: :controller do
+  let(:conference) { create(:conference) }
+  let(:conference_with_open_registration) { create(:conference, registration_period: open_registration_period) }
+  let(:open_registration_period) { create(:registration_period, start_date: Date.current - 6.days) }
```

Comment:
Switch these two lines as you use :open_registration_period to create the :conference_with_open_registration